syz-manager/manager.go

```diff
@@ -1019,7 +1019,7 @@ func (mgr *Manager) minimizeCorpus() {
 	for key := range mgr.corpusDB.Records {
 		_, ok1 := mgr.corpus[key]
 		_, ok2 := mgr.disabledHashes[key]
-		if !ok1 && !ok2 {
+		if !ok1 && !ok2 && !mgr.cfg.PreserveProgs {
```

Comment:
Is this correct? Doesn't this disable minimization when PreserveProgs is set?

Reply:
Yes, that was a bug. I moved the check to the other place.